cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
It's no problem for CAS to have a focused XML file for these mappings (on the grand scheme of things), but a huge advantage (IMHO) for deployers that they don't see this "plumbing" in their deployer config, so it's narrowly focused on their custom CAS artifacts

On Fri, Apr 8, 2016 at 06:56, Misagh Moayyed <notifications@github.com> wrote:
In cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml [https://github.com/Jasig/cas/pull/1676#discussion_r59009020] :

> @@ -0,0 +1,29 @@
> +<?xml version="1.0" encoding="UTF-8"?>
> +<beans xmlns="http://www.springframework.org/schema/beans"
> +       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"


Let me clarify: I understand why you chose a separate file. My concern is:

 1. Should this be something an overlay could do, rather than CAS?
 2. Yet another XML file?
 3. Do we ever see the need for a deployer to overlay this new XML file? If so, why not keep it in the deployer, and limit the configuration to one xml file only?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/Jasig/cas/pull/1676/files/9c8a4b49aee43d91c0113ec6f61045218cf9e71f#r59009020]